templates/etc/apache2/conf-available/global-server-name.conf

```diff
@@ -1 +1,3 @@
 ServerName "localdev"
+LogLevel debug
+LogLevel rewrite:debug
```

Comment:
What's the rationale for increasing the log output here? Is this a change we'll want in production?

Reply:
I assumed this file is not part of the prod deployment since the first line contains `localdev`. These logs might not be necessary, but I did find them useful during the devenv setup changes

Reply:
This raised an interesting question; the file does indeed name the server "localdev", but most things that are differentiated by environment in this repo would have separate files for each env,  and this does not. It turns out this file has no effect on either the deployed instances or the local VM, because of a bug in the configuration script. It tries via `cp "${TEMPLATES_PATH}/etc/apache2/conf-available/*" /etc/apache2/conf-available/` to move this and a few other configuration files into the appropriate apache directory, but it fails because the double quotes prevent the * from expanding. `cp "${TEMPLATES_PATH}/etc/apache2/conf-available"/* /etc/apache2/conf-available/` would copy the files successfully, but I don't know that we want to make that change because this would result in them applying to all environments, and this one at least seems to have been meant only for local.
My inclination is to remove the log changes, and create a ticket in the potholes sprint to either remove these config files or start actually using them (presumably in all environments).